plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeColumnHandle.java

```diff
@@ -51,16 +51,28 @@
     private final String name;
     private final Type type;
     private final DeltaLakeColumnType columnType;
+    private final Optional<String> comment;
+
+    @Deprecated
+    public DeltaLakeColumnHandle(
```

Comment:
Can you please remove the usage of this constructor from the current codebase of `trino-delta-lake` module? I still found `2` usages of it.

Reply:
Actually, there're a lot of usages more than 2. That is why I marked `@Deprecated` instead of migrating them.

Reply:
DLCH is used as key in maps, comment is part of its equality.
We unfortunately  need to update all usages appropriately, otherwise we cannot reason about the correctness of the codebase.

I would very much prefer not adding comment here, to the table handle.
It bite us a few times in Iceberg, so i would prefer to see if we can transport the comment to ColumnMetadata somehow else.